cypress/utils/utils.ts

```diff
@@ -392,9 +392,18 @@ export function notExists(value: string, tableSelector = appTable): void {
     });
 }
 
-export function selectFilter(filterName: string): void {
-    cy.get("#filtered-by").click();
-    clickWithinByText('div[class="pf-v5-c-menu__content"]', "button", filterName);
+export function selectFilter(filterName: string, eq = 0): void {
+    if (eq === 0) {
+        cy.get("#filtered-by").click();
+        clickWithinByText('div[class="pf-v5-c-menu__content"]', "button", filterName);
+        return;
+    }
+    cy.get("div.pf-m-filter-group")
+        .eq(eq)
+        .within(() => {
+            cy.get("#filtered-by").click();
```

Comment:
You can add this locator to common.view since it is reused.

Reply:
Thanks! solved in b0815f61959e19d40bf1d79de15e6c95d3ed84ac